script/playbackstrategy/modifiers/live/seekable.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @fileOverview Requirejs module containing device modifier for live playback
+ * with support level Seekable
+ * @preserve Copyright (c) 2013-present British Broadcasting Corporation. All rights reserved.
+ * @license See https://github.com/bbc/tal/blob/master/LICENSE for full licence
```

Comment:
Does all this stuff need removing?

Reply:
Removed with recent [commit](https://github.com/bbc/bigscreen-player/pull/29/commits/68f349789e7c0bc00dffb35ea07b722a8f8518c4)